CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 This file is used to list changes made in each version of the `chef_handler_sns` cookbook.
 
+## v3.1.0 (2017-05-05)
```

Comment:
Maybe put *unreleased* here. Mainly because probably I going to make more changes before releasing and, second, because probably it is going to be a major bump (I have not decided it yet).